docs/Esport Protocol V1/Tech integration/api-touchpoints.md

```diff
@@ -408,13 +429,12 @@ To verify an OTP, the client must send a JSON object containing the following in
 
 # API Documentation: Rewards Information API
 
-## Endpoint: GET `/api/v1/rewards?token={token}`
+## Endpoint: GET `/fe/esports/rewards/claim`
 This endpoint is designed to provide users with information about the rewards they have earned. It's a key feature for platforms that offer reward-based incentives or achievements.
```

Comment:
Can we change this sentence to **This endpoint is designed to provide users with information about the claimable rewards they have earned**